app/react/Documents/components/specs/DocumentSidePanel.spec.js

```diff
@@ -108,6 +109,27 @@ describe('DocumentSidePanel', () => {
         ).toBe('metadata');
       });
     });
+
+    describe('when doc passed has toc', () => {
+      const toc = ['title1', 'title2', 'title3'];
+
+      it('if is a new entity should set toc of the loaded file', () => {
+        props.doc = Immutable.fromJS({ metadata: [], attachments: [], type: 'entity' });
+        props.file = { toc, pdfInfo: {} };
+        props.tab = 'toc';
+        render();
+        expect(component.find({ toc })).toHaveLength(1);
```

Comment:
The test says that "it should set the ToC of the loaded file"... yet the actual expect here only tests for length.  I think that the description is correct, but the tests needs to proof it is getting the "correct" ToC.